hpxml_version_translator/converter.py

```diff
@@ -781,15 +781,15 @@ def get_pre_post_from_building_id(building_id):
         if hasattr(this_attic, "AtticRoofInsulation"):
             roof_insulation = deepcopy(this_attic.AtticRoofInsulation)
             roof_insulation.tag = f"{{{hpxml3_ns}}}Insulation"
-            roof_idref = this_attic.AttachedToRoof.attrib["idref"]
             try:
+                roof_idref = this_attic.AttachedToRoof.attrib["idref"]
```

Comment:
If there is no `AttachedToRoof` element, it was throwing a python error. Now it's caught as part of the `except` below.